docs/source/stimuli.md

```diff
@@ -0,0 +1,53 @@
+# Preparing your stimuli
+
+### Audio and video files
+
+Most experiments will involve using audio and/or video files! You are responsible for hosting these somewhere (contact MIT if you need help finding a place to put them).
+
+For basic editing of audio files, if you don't already have a system in place, we highly recommend [Audacity](http://www.audacityteam.org/). You can create many "tracks" or select portions of a longer recording using labels, and export them all at once; you can easily adjust volume so it's similar across your stimuli; and the simple "noise reduction" filter works well.
+
+### File formats
+
+To have your media play properly across various web browsers, you will generally need to provide multiple file formats. For a comprehensive overview of this topic, see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats). 
+
+MIT's standard practice is to provide mp3 and ogg formats for video, and webm and mp4 (H.264 video codec + AAC audio codec) for video, to cover modern browsers. The easiest way to create the appropriate files, especially if you have a lot to convert, is to use the command-line tool [ffmpeg](https://ffmpeg.org/). It's a bit of a pain to get used to, but then you can do almost anything you might want to with audio and video files. 
```

Comment:
Sorry, old habit from my grad school days. :) 

(there actually are copyediting agencies for the sciences, but from what I hear they don't pay the bills nearly as much as you'd think....)